src/UserInfoHandler.ts

```diff
@@ -0,0 +1,42 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { Logger } from "@aws-lambda-powertools/logger";
+import { Metrics } from "@aws-lambda-powertools/metrics";
+import { Response } from "./utils/Response";
+import { ClaimedIdRequestProcessor } from "./services/ClaimedIdRequestProcessor";
+import { ResourcesEnum } from "./models/enums/ResourcesEnum";
+import { AppError } from "./utils/AppError";
+import { HttpCodesEnum } from "./utils/HttpCodesEnum";
+import { UserInfoRequestProcessor } from "./services/UserInfoRequestProcessor";
+
+const POWERTOOLS_METRICS_NAMESPACE = process.env.POWERTOOLS_METRICS_NAMESPACE ? process.env.POWERTOOLS_METRICS_NAMESPACE : "CIC-CRI";
+const POWERTOOLS_LOG_LEVEL = process.env.POWERTOOLS_LOG_LEVEL ? process.env.POWERTOOLS_LOG_LEVEL : "DEBUG";
+
+const logger = new Logger({
+    logLevel: POWERTOOLS_LOG_LEVEL,
+    serviceName: "UserInfoHandler",
+});
+
+const metrics = new Metrics({ namespace: POWERTOOLS_METRICS_NAMESPACE });
+
+export const lambdaHandler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
```

Comment:
this needs to be modified to implement LambdaInterface and addition of annotation as in [here](https://github.com/alphagov/di-ipv-cri-cic-api/blob/4c48907bf87dbcdab28ec213ed0917d226bed3a5/src/ClaimedIdentityHandler.ts#L24)